### PR TITLE
Update WAFS before EVSv1 code freezing

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
1. Copy stat files only when SENDCOM=YES
2. Remove FIXevs assignment in stat driver
3. Remove lines of verbosity from wafs parm files
4. Update contact email address

## Pull Request Testing ##

- [ x] Describe testing already performed for this Pull Request:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ x] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [x ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by PerryShafran-NOAA** [comment]
Hi, @YaliMao-NOAA - I have picked up your PR and I am now running the stats test.

**Comment by PerryShafran-NOAA** [comment]
Hi, @YaliMao-NOAA ,

The stats job is complete.

Please see the .o file here:

/lfs/h2/emc/vpppg/noscrub/perry.shafran/pr245test/EVS/dev/drivers/scripts/wafs/stats/jevs_wafs_stats.o81979709

and your final stats output is here:

/lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/wafs/wafs.20230914

Have a look and let me know if all is good.

**Comment by YaliMao-NOAA** [comment]
@PerryShafran-NOAA As expected, the outputs are the same as what I ran before the PR. So they are good.

**Comment by PerryShafran-NOAA** [comment]
Super!  Let me review the code.

File: jobs/wafs/stats/JEVS_WAFS_ATMOS_STATS [hunk 1]
```diff
@@ -132,10 +145,16 @@ for resolution in $resolutions ; do
...
+	awk '!seen[$0]++' $finalfile > $STATSOUTfinal/$finalfile
     fi
 done
 
+if [ $SENDCOM = YES ] ; then
+    mv $STATSOUTfinal/* $COMOUTfinal/.
+    # COMOUTsmall
+    mv $STATSOUTsmall/* $COMOUTsmall/.
+fi
+
```
Comment (PerryShafran-NOAA) [inline jobs/wafs/stats/JEVS_WAFS_ATMOS_STATS]: I'm not entirely sure if this SENDCOM stuff can be in the J-job or needs to be in the exevs script.  @AliciaBentley-NOAA what do you think here?

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA @YaliMao-NOAA 
SENDCOM should be defined in the J-job ([which it is on line 12](https://github.com/NOAA-EMC/EVS/pull/245/files/6c30444f2cc918021bc0ddeeec05365391d74037#diff-f220eb84e38c80e7efd512d715353b4a1dbfa4d3a35db2a89354947319f94869R12)), but it should not be used in the J-job. It should be used inside the exevs scripts called [on either line 115 or line 117](https://github.com/NOAA-EMC/EVS/pull/245/files/6c30444f2cc918021bc0ddeeec05365391d74037#diff-f220eb84e38c80e7efd512d715353b4a1dbfa4d3a35db2a89354947319f94869R115-R117). Thanks!

File: jobs/wafs/stats/JEVS_WAFS_ATMOS_STATS (hunk 1 above)
Comment (PerryShafran-NOAA) [reply jobs/wafs/stats/JEVS_WAFS_ATMOS_STATS]: @YaliMao-NOAA Can you move the SENDCOM check from the J-job to the exevs script?

Comment (PerryShafran-NOAA) [reply jobs/wafs/stats/JEVS_WAFS_ATMOS_STATS]: @YaliMao-NOAA this needs to be resolved now.  Code freeze is happening now, so this issue needs to be resolved.  Can you tend to it?

**Comment by PerryShafran-NOAA** [comment]
@YaliMao-NOAA - thank you for this update.  Let me do a test of this change and then have you look at it for me.

**Comment by PerryShafran-NOAA** [comment]
@YaliMao-NOAA - just wanting to be sure; your merge of the remote-tracking branch, that is the NOAA-EMC develop and not just the develop from your own fork; unless that was updated with the NOAA-EMC develop also.

**Comment by PerryShafran-NOAA** [comment]
@YaliMao-NOAA Looks like your change was successful.

Please look at the .o file here: /lfs/h2/emc/vpppg/noscrub/perry.shafran/pr245testa/EVS/dev/drivers/scripts/wafs/stats/jevs_wafs_stats.o82244562

And the output is here: /lfs/h2/emc/vpppg/noscrub/perry.shafran/evs/v1.0/stats/wafs/wafs.20230917

Please validate for me.

Thanks!

Perry

**Comment by YaliMao-NOAA** [comment]
@PerryShafran-NOAA Both look good. The log file looks good to me and the stat outputs are the same as my test result.

**Comment by PerryShafran-NOAA** [comment]
Thank you!  The rest of the code items were already looked at, so I will go ahead and merge now.

**Review by PerryShafran-NOAA — APPROVED** [review]

**Pull request merged**